prepare_cloud_image.py

```diff
@@ -140,8 +140,8 @@ def run_qemu(self):
             '-m', '512',
             '-netdev', 'user,id=user',
             '-device', 'virtio-net-pci,netdev=user',
-            '-hda', str(self.disk_img),
-            '-hdb', str(self.cloud_init_img),
+            '-drive', 'index=0,media=disk,file=' + str(self.disk_img),
+            '-drive', 'index=1,media=disk,format=raw,file=' + str(self.cloud_init_img),
```

Comment:
I assume these lines are equivalent. Why do you want to change them?

Reply:
Yes, they're equivalent.

`format=raw` is needed to silence a warning about `qemu` guessing the format type. There's no way to set a format with the `-hda` option.

I only needed to change line 144, true, but it's more consistent this way.

Reply:
Ah, makes sense, ok.